src/PartitionService.ts

```diff
@@ -101,7 +90,79 @@ export class PartitionService {
         return Math.abs(partitionHash) % this.partitionCount;
     }
 
-    getPartitionCount(): number {
+    /**
+     * If partition table is not fetched yet, this method returns zero
+     *
+     * @return the partition count
+     */
+    public getPartitionCount(): number {
         return this.partitionCount;
     }
+
+    /**
+     * Resets the partition table to initial state.
+     */
+    public reset(): void {
```

Comment:
nit: this method seems to be unused. Is it on purpose?

Reply:
This method is used as a part of the Blue/Green failover logic in the Java side. I probably forget to remove it. So, I am removing it.